ZelBack/src/services/fluxNetworkHelper.js

```diff
@@ -1328,6 +1331,22 @@ async function purgeUFW() {
   }
 }
 
+/**
+ * This fix a docker security issue where docker containers can access host network, for example to create port forwarding on hosts
+ */
+async function removeDockerContainerAccessToHost() {
```

Comment:
Can you explain it again why this is needed?

Reply:
this fix a vulnerability where containers can access ufw/port forwading/upnp by having access to the host subnet.
vulnarability was raised by a community member.
example, go to a upnp node, manage any app, and on execute commands execute: curl http://192.168.1.1:2189/rootDesk.xml

you might need to install curl first. 

we might comment this to do more testing before we release it.

Reply:
it's commented for more testing